bagel/download.py

```diff
@@ -11,6 +9,21 @@
 def get_metadata():
     # download metadata from google sheet and save it as a csv
     df = pd.read_csv(
-        URL, usecols=range(0, 25), converters={22: lambda x: x.replace("\n", " ")}
+        URL,
+        usecols=range(0, 25),
+        converters={
```

Comment:
Interesting, I initially thought data types should be specified in `dtypes` argument. Looks like `converters` is very flexible way to configure parsing.